R/Samples-validity.R

```diff
@@ -0,0 +1,32 @@
+#' @include McmcOptions-methods.R
+NULL
+
+# v_samples ----
+
+#' Internal Helper Functions for Validation of [`Samples`] Objects
+#'
+#' @description `r lifecycle::badge("stable")`
```

Comment:
so for any internal functions or classes we should not use lifecycle badges. they are reserved for the user API, i.e. exported stuff